markdown_anki_decks/cli.py

```diff
@@ -48,6 +48,11 @@ def convertMarkdown(
         "--prefix",
         help="Can be used to make your markdown decks part of a single subdeck. Anki uses `::` to indicate sub decks. `markdown-decks::` could be used to make all generated decks part of a single root deck `markdown-decks`",
     ),
+    filetype: str = typer.Option(
```

Comment:
Can we change the name of this parameter to `extension` with a short option  of `--ext`? That would match the API exposed by other similar programs. For example, see [eslint](https://eslint.org/docs/0.24.1/user-guide/command-line-interface#--ext).

Can we allow the user to pass multiple extensions instead of just a single string? See the page [multiple cli options](https://typer.tiangolo.com/tutorial/multiple-values/multiple-options/) in the typer docs for instructions on how to do this. We would have to change the default value to `["md"]` and the type to `Optional[List[str]]`.